app/assets/stylesheets/_shirts.scss

```diff
@@ -0,0 +1,21 @@
+div.shirt-banner {
+  margin-bottom: $base-spacing;
+  overflow-x: hidden;
+  white-space: nowrap;
+
+  a {
+    @include animation(marquee 20.0s linear infinite);
+    color: white;
+    display: inline-block;
+    font-style: italic;
+    font-weight: bold;
+    padding-left: 100%;
+    text-decoration: none;
```

Comment:
Eh... it doesn't have an underline (which I think makes it look slightly better), but it does scroll across the screen and it's bold and italic. It's not exactly subtle, but no, there is nothing that explicitly says "I'm a link" until you hover over it or click it. I'm feel about a 2/5 on this so if you feel strongly I'll add it!